packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -228,17 +230,15 @@ function check_scan_file()
 end
 
 -- Read scan lock file
-function check_lock_file()
-    local file = io.open("/etc/first_run", "r")
-    if(file == nil) then
-        return false
-    end
-    return true
+function fbw.check_lock()
+  uci_cursor = uci.cursor()
+  return uci_cursor:get("lime", "first_run") or true
 end
 
 -- Remove lock file
-function remove_lock_file()
-    utils.execute("rm /etc/first_run")
+function fbw.remove_lock()
+  uci_cursor = uci.cursor()
+  uci_cursor:set("lime", "first_run", true)
```

Comment:
I am not so sure that a boolean value can be stored through uci. `lime.config` has a method to set and get boolean values.

Reply:
It can use boolean. check https://github.com/libremesh/lime-packages/blob/master/packages/lime-docs/files/lime-example

Reply:
That is used through `lime.config` and uci reads it like a string not a boolean. 

The other thing is that I think there is something odd with the logic now. Removing the lock is seting "first_run" to true ? It is not very clear. I would use something like "fbw_configured=1" or something more closer to "not_first_run=1".

Reply:
when I do `uci set lime.first_run=true; uci commit` I get:
```
root@ql-roxa:~# grep first_run /etc/config/lime 
config true 'first_run'
```

When I read that value from lua:
```
# cat test.lua 
local uci = require("uci")
uci_cursor = uci.cursor()
first_run=uci_cursor:get("lime", "first_run")
print(first_run)
root@ql-roxa:~# lua test.lua 
true
```

Reply:
that way you are creating a section of type "true", options must be something like lime.section_name.option_name  why aren't you using lime.config library? That is what should be use to edit /etc/config/lime